src/scripts/browserDeviceProfile.js

```diff
@@ -235,6 +235,7 @@ function supportsHlg(options) {
 function supportsDolbyVision(options) {
     return options.supportsDolbyVision ?? (false // eslint-disable-line sonarjs/no-redundant-boolean
             || browser.safari && ((browser.iOS && browser.iOSVersion >= 13) || browser.osx)
+            || browser.edgeChromium && browser.xboxOne && window.NativeShell
```

Comment:
For apps, pass `supportsDolbyVision: true` in the options to the profile builder from the NativeShell implementation.

Reply:
Good point. Thanks. Added in https://github.com/jellyfin/jellyfin-uwp/pull/44